.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,8 +8,10 @@ out the development guide https://matplotlib.org/devdocs/devel/index.html
 (Why is this change required?  What problem does it solve?) and link to relevant
 issues and PRs.
 
-Also please summarize the changes in the title, for example "Raise ValueError on
-non-numeric input to set_xlim" and avoid non-descriptive titles such as "Addresses
+Additionally, please explain in 1-4 sentences your implementation choices.
```

Comment:
This is what I am thinking, because I agree, these should be explicit, and having them in parenthesis takes away from its importance. 

## PR summary
Please provide at least 1-2 sentences, per question, describing the pull request in detail and links to relevant issues and PRs:

- Why this change is necessary? 
- What problem it solves? 
- Why this solution is implemented in this manner?

Additionally, please summarize the changes in the title, for example "Raise ValueError on
non-numeric input to set_xlim" and avoid non-descriptive titles such as "Addresses
issue #8576".